pkg/apis/forklift/v1beta1/provider.go

```diff
@@ -75,6 +75,9 @@ type ProviderSpec struct {
 //
 // ProviderStatus defines the observed state of Provider
 type ProviderStatus struct {
+	// Current life cycle phase of the provider.
+	// +optional
+	Phase string `json:"phase,omitempty"`
```

Comment:
Are you able to access this value also from `kubectl`? If I remember correctly you might need to add the `kubebuilder:printcolumn:name` right above the `Provider struct`
Something like 
```
// +kubebuilder:printcolumn:name="PHASE",type="string",JSONPath=".status.phase"
```

Reply:
Thanks, missed that ( didn't know you can do that :-) )
Added the phase column before the conditions

```bash
oc get providers
NAME   TYPE    PHASE   READY   CONNECTED   INVENTORY   URL                                                          AGE
t1     ovirt   Ready   True    True        True        https://vm-10-173.lab.eng.tlv2.redhat.com/ovirt-engine/api   6m48s
```